File: couch/ringo/create/couch_create.js
#! /usr/bin/ringo
// ------------------------------------------------------------------
//
//	couch_create.js
//
//						Dec/19/2014
//
// ------------------------------------------------------------------
var text_manipulate = require ('/var/www/data_base/common/ringo_common/text_manipulate.js');
// ------------------------------------------------------------------
print	("*** 開始 ***");
var client= require ("ringo/httpclient");
var url='http://localhost:5984/nagano';

client.del (url);
client.put (url);

var dict_aa = data_prepare_proc ();

for (var key in dict_aa)
	{
	print (key);
	var unit_aa = dict_aa[key];
	var out_str = JSON.stringify (unit_aa);
	var url_target = url + "/" + key;
	client.put (url_target,out_str);
	}

print	("*** 終了 ***");
// ------------------------------------------------------------------
function data_prepare_proc ()
{
	var dict_aa = new Object ();

	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2021','長野',73865,'1953-8-12');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2022','松本',24157,'1953-11-18');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2023','上田',68491,'1953-5-2');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2024','小諸',54837,'1953-6-22');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2025','岡谷',47352,'1953-8-14');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2026','塩尻',12564,'1953-9-12');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2027','茅野',93871,'1953-3-21');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2028','飯田',58296,'1953-7-26');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2029','中野',28457,'1953-10-2');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2030','諏訪',45873,'1953-12-20');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2031','駒ヶ根',91857,'1953-2-7');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2032','佐久',23954,'1953-5-17');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2033','伊那',81675,'1953-6-19');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2034','千曲',72548,'1953-10-14');


	return	dict_aa;
}

// ------------------------------------------------------------------

File: couch/jython/create/couch_create.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	create/couch_create.py
#
#					Jun/24/2014
#
# ----------------------------------------------------------------
import	sys
reload(sys)
sys.setdefaultencoding('utf-8')
import	string
import	json
#
sys.path.append ('/var/www/data_base/common/python_common')
from text_manipulate import dict_append_proc
#
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_get_uri import rest_put_proc
from jython_get_uri import rest_delete_proc
#
# ----------------------------------------------------------------
def	data_prepare_proc ():
	dict_aa = {}
	dict_aa = dict_append_proc (dict_aa,'t2021',u'長野',91237,'2004-10-17')
#
	dict_aa = dict_append_proc (dict_aa,'t2022',u'松本',36584,'2004-8-15')
#
	dict_aa = dict_append_proc (dict_aa,'t2023',u'上田',57412,'2004-6-12')
	dict_aa = dict_append_proc (dict_aa,'t2024',u'小諸',61538,'2004-7-21')
	dict_aa = dict_append_proc (dict_aa,'t2025',u'岡谷',72896,'2004-4-19')
	dict_aa = dict_append_proc (dict_aa,'t2026',u'塩尻',84123,'2004-11-14')
	dict_aa = dict_append_proc (dict_aa,'t2027',u'茅野',96357,'2004-5-24')
	dict_aa = dict_append_proc (dict_aa,'t2028',u'飯田',53172,'2004-2-12')
	dict_aa = dict_append_proc (dict_aa,'t2029',u'中野',64987,'2004-11-14')
	dict_aa = dict_append_proc (dict_aa,'t2030',u'諏訪',82154,'2004-7-04')
	dict_aa = dict_append_proc (dict_aa,'t2031',u'駒ヶ根',72681,'2004-9-17')
	dict_aa = dict_append_proc (dict_aa,'t2032',u'佐久',28743,'2004-7-20')
	dict_aa = dict_append_proc (dict_aa,'t2033',u'伊那',12759,'2004-3-12')
	dict_aa = dict_append_proc (dict_aa,'t2034',u'千曲',62938,'2004-8-17')
#
	return	dict_aa
#
# ----------------------------------------------------------------
print ("*** 開始 ***")
#
host = "localhost:5984"
collection = "/nagano"
url_collection = 'http://' + host + collection
#
rest_delete_proc (url_collection)
rest_put_proc (url_collection,"{}","text/json")
#
dict_aa = data_prepare_proc ()
#
for key in dict_aa.keys():
	str_json = json.dumps (dict_aa[key])
	url_target = url_collection + "/" + key
	rest_put_proc (url_target,str_json,"text/json")
#
print ("*** 終了 ***")
# ----------------------------------------------------------------
